nio/events/account_data.py

```diff
@@ -67,6 +69,28 @@ def from_dict(cls, event_dict):
         )
 
 
+@attr.s
+class TagEvent(AccountDataEvent):
+    """Tag event.
+
+    Informs the client tags of a room.
+
+    Attributes:
+        tags (Dict): The tags of the room
```

Comment:
Could you describe what the format of the key/values of the dictionary is? It isn't clear without looking for the spec and ideally we would like to avoid trips to the spec if possible.